lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -236,6 +238,19 @@ final class ComicFormViewController: UIViewController, AlertDisplaying {
         releaseDateTextField.inputAccessoryView = releaseDateToolBar
     }
 
+    private func checkReleaseYear(completion: () -> Void) {
```

Comment:
This doesn't need to have a completion, you can just have it return a Bool and if it's false show the error there